CalibCalorimetry/HcalAlgos/src/HcalLogicalMapGenerator.cc

```diff
@@ -64,18 +64,32 @@ HcalLogicalMap HcalLogicalMapGenerator::createMap(const HcalTopology* topo, unsi
   uint32_t illegal_value = 0;
   for (int i = 0; i <= HcalElectronicsId::maxLinearIndex; i++)
     LinearIndex2Entry.push_back(illegal_value);
+  HbHash2Entry.reserve(2 * HbHalf);
+
```

Comment:
can the additional line break be avoided?

Reply:
I would follow Kevin comment.

Reply:
... or at least have it above the line with "reserve", rather than below.
Out of curiosity: why there is no reserve for the vector "LinearIndex2Entry"?

Reply:
Looks like the check does not understand `<=`.

For example, in this simple test
```c++
int main(void) {
  constexpr int size = 10;

  std::vector<int> v;
  for (int i = 0; i < size; ++i)
    v.push_back(i);

  std::vector<int> w;
  for (int i = 0; i <= size; ++i)
    w.push_back(i);

  return 0;
}
```
it will add a reserve for `v` and not for `w`.

Reply:
why wouldn't we go for a one-liner here?
```c++
    // Create a vector of size n with 
    // all values as 10. 
    vector<int> vect(n, 10); 
```

Reply:
I agree that would be better - looks like `clang-tidy` is just not smart enough about it.